### PR TITLE
[plugin.video.nrk] v 4.8.0

### PR DESCRIPTION
**Pull request by andrea99boban**
### Checklist:

- [ x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [ x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by basrieter** [comment]
@tamland are you ok with the PR to your add-on?

**Comment by basrieter** [comment]
Please fix the errors from the checker that ran. And please the guidelines for add-ons here: https://kodi.wiki/view/Add-on_rules

File: xbmc-addon-nrk-master/addon.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <language>no</language>
+    <summary lang="en">Watch NRK Web TV</summary>
+    <summary lang="no">Se NRK Nett-TV</summary>
+    <summary lang="it">Guarda NRK (in Norvegia)</summary>
+    <description lang="en">NRK Web TV gives you the opportunity to watch live TV as well as recordings of many shows.</description>
+    <description lang="no">NRK Nett-TV gir deg muligheten til å se mange av NRKs programmer både direkte og i opptak.</description>
+    <description lang="it">Questo addon per NRK rende possibile la visione di molti programmi di NRK sia in diretta che in differita.</description>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
+    <forum>https://forum.kodi.tv/showthread.php?tid=52824</forum>
+    <source>https://github.com/tamland/xbmc-addon-nrk</source>
```
Comment (basrieter) [inline xbmc-addon-nrk-master/addon.xml:25]: Is @tamland still the maintainer? Because these sources are not in his repo?

Comment (andrea99boban) [reply xbmc-addon-nrk-master/addon.xml:25]: Hi, i don't know - he seems to be absent for some time, it was a bit awkward to see Nexus without NRK, so i'm trying to fill in. If he gets back to it he can also get my mods if he like them, and add to his main, no issue on my side. I did a 0.1% to just make it work under Nexus. All the work is his, obviously.

Comment (tamland) [reply xbmc-addon-nrk-master/addon.xml:25]: I'm not actively maintaining it and have left it up to others/the community. I don't know if those people are also inactive now or not, however there appears to be activity in the repo 2 week ago? https://github.com/tamland/xbmc-addon-nrk/commits/master

Comment (basrieter) [reply xbmc-addon-nrk-master/addon.xml:25]: So would it make sense to do this PR there and and then a proper PR from [https://github.com/tamland/xbmc-addon-nrk](https://github.com/tamland/xbmc-addon-nrk/commits/master) to Kodi?

Comment (tamland) [reply xbmc-addon-nrk-master/addon.xml:25]: I have merged the open PR at least https://github.com/tamland/xbmc-addon-nrk/pull/73

Comment (andrea99boban) [reply xbmc-addon-nrk-master/addon.xml:25]: happy to have helped

**Pull request closed without merging**